source/chapters/advanced_topics.rst

```diff
@@ -233,3 +233,32 @@ using sends for effects. This gives Mixxx access to the extensive collection of
 Make sure the correct multichannel audio interface has been selected in Jack
 (Jack settings visible bottom left). Note that Mixxx possibly labels its Jack
 ports as :guilabel:`Portaudio`.
+
+.. _advanced-migrate-settings:
+
+Migrate your Mixxx library and settings to a new computer
+=========================================================
+
+If you move to a new computer, or to another operating system on the same computer, you can take your existing Mixxx setup with you. This includes your track library, settings, controller mappings and broadcast profiles.
+
+You don't necessarily need to be able to start your previous Mixxx installation, all you need is your :ref:`settings directory <appendix-settings-files>` and the music directories you imported into your Mixxx library via :menuselection:`Preferences --> Library`.
+
+Preparation
+-----------
+
+* If you still have access to your Mixxx installation open Mixxx, go to :menuselection:`Preferences --> Library` and uncheck :guilabel:`Rescan on startup`. If you are not able to start Mixxx anymore, e.g. if you only managed to recover your settings directory, you can do this manually in the settings file: open :file:`mixxx.cfg` with a text editor, delete the line with :file:`RescanOnStartup` and save the file.
+* Make a copy of both the Mixxx settings directory and your music directories. Put all in a save place. Do not restructure your music directories!
+* Install you new operating system, get your new PC ready.
+* Get the current stable Mixxx version from `the official download page <https://mixxx.org/download/>`_.
+* Install Mixxx, though don't start it, yet.
```

Comment:
same here, "though don't start it, yet." should probably be in bold otherwise people will struggle with the remaining steps.